hasura.planx.uk/migrations/1703240593405_squashed/up.sql

```diff
@@ -0,0 +1,30 @@
+CREATE TABLE "public"."team_themes" (
+    "id" serial NOT NULL,
+    "team_id" integer NOT NULL,
+    "primary_colour" text NOT NULL DEFAULT '#0010A4',
+    "secondary_colour" text,
```

Comment:
I'll check with @ianjon3s in the new year how we should handle defaults for this column.